**В ящике имеется 15 деталей, из которых 9 окрашены. Рабочий случайным образом извлекает 3 детали.**

**Какова вероятность того, что все извлеченные детали окрашены?**

Общее количество исходов будет определяться количеством сочетаний 3 деталей из 15,

 а количество благоприятных исходов - количеством сочетаний 3-х окрашенных деталей из 9.

Вводим функцию расчета исходов

In [1]:
from math import factorial
def combinations(n, k):
    return int(factorial(n) / (factorial(k) * factorial(n - k)))

Расчитываем количество благоприятных исходов - m, общее количество исходов - n  и вероятность извлечения окрашенных деталей

In [2]:
m=combinations(9,3)
n=combinations(15,3)
P=m/n
print(f'P(3 из 3-х окрашены) = {round(P,4)}')

P(3 из 3-х окрашены) = 0.1846


Ответ: вероятность, что рабочий вытащит 3 окрашенные детали составляет P=0.1846 или примерно 18.46%.